web/client/configs/localConfig.json

```diff
@@ -10,7 +10,8 @@
           "https://gs-stable.geo-solutions.it/geoserver",
           "https://gs-stable.geo-solutions.it:443/geoserver",
           "http://gs-stable.geo-solutions.it/geoserver",
-          "http://gs-stable.geo-solutions.it:443/geoserver"
+          "http://gs-stable.geo-solutions.it:443/geoserver",
+          "https://georchestra.geo-solutions.it/geoserver"
```

Comment:
i don't think we want to include this geoserver among the cors urls in the official mapstore repo
could you explain the cause of the issue and why you ended up with this fix?

Reply:
@tonym3nsa please reply for the above

Reply:
@MV88 sorry, I must have missed this. The 3D map was unable to render as described [here](https://github.com/geosolutions-it/MapStore2/issues/7661). From my investigations, it seems like an issue caused by CORS - that's why I added the URLs there

Reply:
i'll better check this tomorrow, but we do not need to include every geoserver url for every client in the official repo
eventually you include it in the dedicated project if needed. otherwise it may be a problem of the proxy or something else

@offtherailz what do you think?

Reply:
I think we should use the internal proxy in a transparent way as much as possible. 
Even if CORS is a useful optimization, we can not rely on it. The old version worked without this setting, so we need to keep the same functionality. 
Try to understand how the old worked, how this functionality disappeared during the updated, and how to restore it.

Reply:
@MV88 @offtherailz Well noted